docs/source/futures.rst

```diff
@@ -799,25 +809,30 @@ Example: Parameter Server
    client = Client(processes=False)
 
    class ParameterServer:
-       def __init__(self):
-           self.data = dict()
-
-       def put(self, key, value):
-           self.data[key] = value
+      def __init__(self):
+          self.data = dict()
 
-       def get(self, key):
-           return self.data[key]
+      def put(self, key, value):
+          self.data[key] = value
 
-   ps_future = client.submit(ParameterServer, actor=True)
-   ps = ps_future.result()
+      def get(self, key):
+          return self.data[key]
 
-   ps.put('parameters', np.random.random(1000))
+   def train(params, lr=0.1):
+      grad = 2 * (params - 2)  # gradient of (params - 2)**2
+      new_params = params - lr * grad
+      return new_params
 
-   def train(batch, ps):
-       params = ps.get('parameters')
+    ps_future = client.submit(ParameterServer, actor=True)
+    ps = ps_future.result()
 
-   for batch in batches:
+    ps.put('parameters', np.random.random(1000))
+    for k in range(20):
+       params = ps.get('parameters').result()
+       new_params = train(params)
+       ps.put('parameters', new_params)
```

Comment:
The indentation here seems slightly off

Reply:
Everything here is good to go except for this comment btw

Reply:
Whoops, thanks.